### PR TITLE
[CloudPC]Add CloudPC reports related API

### PR DESCRIPTION
**Pull request by AshleyYangSZ**
Roadmap Id: 15183
CSDL PR: 
https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/6788638
https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/6881283

api.md PR: 
https://microsoftgraph.visualstudio.com/onboarding/_git/onboarding/pullrequest/10831 
https://microsoftgraph.visualstudio.com/onboarding/_git/onboarding/pullrequest/11593

### CODE REVIEW

File: api-reference/beta/resources/cloudpcreports.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+Represents the Windows 365 Cloud PC related reports, includes Windows 365 Cloud PC remote connections related report.
+
+## Methods
+|Method|Return type|Description|
+|:---|:---|:---|
+|[getDailyAggregatedRemoteConnectionReports](../api/cloudpcreports-getdailyaggregatedremoteconnectionreports.md)|Stream|Get the daily aggregated remote connection reports like round trip time, available bandwidth and so on in a given period.|
+|[getRealTimeRemoteConnectionLatency](../api/cloudpcreports-getrealtimeremoteconnectionlatency.md)|Stream|Get the real time remote connection latency reports like current round trip time, available bandwidth and so on of a Cloud PC.|
+|[getRealTimeRemoteConnectionStatus](../api/cloudpcreports-getrealtimeremoteconnectionstatus.md)|Stream|Get the real time remote connection status reports like sign in status, days since last use of a Cloud PC.|
+|[getRemoteConnectionHistoricalReports](../api/cloudpcreports-getremoteconnectionhistoricalreports.md)|Stream|Get a Cloud PC's the remote connection historical reports like signInDateTime, signOutDateTime and usageInHour in a given period.|
```
Comment (RuiHou105) [inline api-reference/beta/resources/cloudpcreports.md:24]:
```suggestion
|[getRemoteConnectionHistoricalReports](../api/cloudpcreports-getremoteconnectionhistoricalreports.md)|Stream|Get a Cloud PC's remote connection historical reports like signInDateTime, signOutDateTime and usageInHour in a given period.|
```

File: api-reference/beta/api/cloudpcreports-getrealtimeremoteconnectionlatency.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+ms.prod: "cloud-pc"
+doc_type: apiPageType
+---
+
+# cloudPcReports: getRealTimeRemoteConnectionLatency
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Get the real time connection latency related information like RTT and etc of Cloud PC.s
```
Comment (RuiHou105) [inline api-reference/beta/api/cloudpcreports-getrealtimeremoteconnectionlatency.md:15]:
```suggestion
Get the real time connection latency related information like RTT and etc of  a Cloud PC.
```

File: api-reference/beta/api/cloudpcreports-getrealtimeremoteconnectionlatency.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
+---
+title: "cloudPcReports: getRealTimeRemoteConnectionLatency"
+description: "Get the real time connection latency related information like RTT and etc of Cloud PC."
```
Comment (RuiHou105) [inline api-reference/beta/api/cloudpcreports-getrealtimeremoteconnectionlatency.md:3]:
```suggestion
description: "Get the real time connection latency related information like RTT and etc of a Cloud PC."
```

**Review by RuiHou105 — CHANGES_REQUESTED** [review]
Please resolve all comments.

**Comment by isvargasmsft** [comment]
Hello @AshleyYangSZ @RuiHou105 would you be so kind to update the roadmap item and add the CSDL PR, please?

**Comment by AshleyYangSZ** [comment]
> Hello @AshleyYangSZ @RuiHou105 would you be so kind to update the roadmap item and add the CSDL PR, please?

Thanks @isvargasmsft, already added.
Roadmap Id: 15183
CSDL PR: https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/6788638
api.md PR: https://microsoftgraph.visualstudio.com/onboarding/_git/onboarding/pullrequest/10831

**Comment by isvargasmsft** [comment]
> > Hello @AshleyYangSZ @RuiHou105 would you be so kind to update the roadmap item and add the CSDL PR, please?
> 
> Thanks @isvargasmsft, already added. Roadmap Id: 15183

Removing the links as those links are confidential. 

Thank you Ashley :)

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 4]
```diff
@@ -2,14 +2,104 @@
...
+                    "Description": "Added the [cloudPcReports](https://docs.microsoft.com/en-us/graph/api/resources/intune-cloudPcReports?view=graph-rest-beta) resource",
+                    "Target": "cloudPcReports"
+                },
+                {
+                    "Id": "a74bf4db-93a3-4a4c-98b0-400bac1e2131",
+                    "ApiChange": "Resource",
+                    "ChangedApiName": "reports",
+                    "ChangeType": "Addition",
+                    "Description": "Added the **reports** relationship to the [virtualEndpoint](https://docs.microsoft.com/en-us/graph/api/resources/intune-virtualEndpoint?view=graph-rest-beta) resource.",
+                    "Target": "virtualEndpoint"
```
Comment (isvargasmsft) [inline changelog/Microsoft.CloudManagedDesktop.json:43]: Fixing entry:
```suggestion
                    "Id": "a74bf4db-93a3-4a4c-98b0-400bac1e2131",
                    "ApiChange": "Relationship",
                    "ChangedApiName": "reports",
                    "ChangeType": "Addition",
                    "Description": "Added the **reports** relationship to the [virtualEndpoint](https://docs.microsoft.com/en-us/graph/api/resources/intune-virtualEndpoint?view=graph-rest-beta) resource.",
                    "Target": "virtualEndpoint"
```

File: api-reference/beta/api/cloudpcreports-post-exportjobs.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).
+
+|Permission type|Permissions (from least to most privileged)|
+|:---|:---|
+|Delegated (work or school account)|**TODO: Provide applicable permissions.**|
+|Delegated (personal Microsoft account)|**TODO: Provide applicable permissions.**|
+|Application|**TODO: Provide applicable permissions.**|
+
```
Comment (isvargasmsft) [inline api-reference/beta/api/cloudpcreports-post-exportjobs.md:25]: Please complete this table. Replace the TO DO placeholders with the corresponding permissions.

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hi @AshleyYangSZ, 

Thank you very much for your work in this PR. Very complete overall. 

Please take a look at a few comments I left:
1. Please clarify if the member 'failed' should be removed from this documentation as it is not included in the CSDL PR. 
2. Please complete the permissions table in the Create exportJobs topic. 

Feel free to ping me if you have questions. 

Always happy to help. 

Isaac

**Comment by AshleyYangSZ** [comment]
> Please resolve all comments.

@RuiHou105 Already resolved please help review again. Thanks

**Comment by AshleyYangSZ** [comment]
> Hi @AshleyYangSZ,
> 
> Thank you very much for your work in this PR. Very complete overall.
> 
> Please take a look at a few comments I left:
> 
> 1. Please clarify if the member 'failed' should be removed from this documentation as it is not included in the CSDL PR.
> 2. Please complete the permissions table in the Create exportJobs topic.
> 
> Feel free to ping me if you have questions.
> 
> Always happy to help.
> 
> Isaac

Hi @isvargasmsft Already resolved the comments, please help review again. Thanks a lot!

File: api-reference/beta/resources/cloudpcexportjob.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+None.
+
+### cloudPcExportJobStatus values
+
+|Member|Description|
+|:---|:---|
+|notStarted|The export job doesn't start yet.|
+|inProgress|The export job is in progress.|
+|completed|The export job is completed with export url returned. |
+|failed|The export job is failed. |
```
[the remark this answers is not in the document]
Comment (AshleyYangSZ) [reply api-reference/beta/resources/cloudpcexportjob.md:47]: @isvargasmsft Thanks, you help find out a mistake here. Actually, failed is already reviewed in another design api.md PR https://microsoftgraph.visualstudio.com/onboarding/_git/onboarding/pullrequest/11593, and I also added in schema with this https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/6881283. So, let's keep it here.

**Review by isvargasmsft — APPROVED** [review]

**Comment by isvargasmsft** [comment]
@AshleyYangSZ approved :)

**Pull request merged**